src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -845,4 +846,57 @@ public void shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled() {
       hasJsonPath("loan.itemStatus", "Checked out"))
     ));
   }
+
+  @Test
+  public void loanDueDateTruncatedOnCheckoutWhenRecallAnywhereInQueue() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource james = usersFixture.james();
+    final IndividualResource rebecca = usersFixture.rebecca();
+    final IndividualResource steve = usersFixture.steve();
+
+    final Period loanPeriod = Period.weeks(4);
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(loanPeriod)
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+      .withRecallsRecallReturnInterval(Period.weeks(2)));
+
+      checkOutFixture.checkOutByBarcode(
+        smallAngryPlanet, steve, now(UTC));
+
+      requestsFixture.placeHoldShelfRequest(
+        smallAngryPlanet, james, now(UTC),
+        requestServicePoint.getId(), "Hold");
```

Comment:
Is the last parameter the patron comments?

Reply:
I don't think so.  If you look at the definition of that method, that seems to describe the type of hold.  Which seems odd considering the name of the method, but that's how it's written.

Reply:
Strange, probably something I did a while ago and needs improving.

It can be confusing, any type of request (hold, recall or page) can be fulfilled to the hold shelf. The method name refers to the fulfilment preference rather than the type. It took me quite a long time to get this to stick in my head.